cmd/handlers.go

```diff
@@ -124,6 +124,8 @@ func initHTTPHandlers(e *echo.Echo, a *App) {
 		g.GET("/api/bounces/:id", pm(hasID(a.GetBounce), "bounces:get"))
 		g.DELETE("/api/bounces", pm(a.DeleteBounces, "bounces:manage"))
 		g.DELETE("/api/bounces/:id", pm(hasID(a.DeleteBounce), "bounces:manage"))
+		g.PUT("/api/bounces/blocklist", pm(a.BlocklistSubscriberBounces, "bounces:manage"))
```

Comment:
There is no need to create new handlers.

`/api/bounces` has `subscriber_id` on each bounce. That can be used to send blocklist requests to the existing handlers.

```
		g.PUT("/api/subscribers/blocklist", pm(a.BlocklistSubscribers, "subscribers:manage"))
		g.PUT("/api/subscribers/:id/blocklist", pm(hasID(a.BlocklistSubscriber), "subscribers:manage"))
```

Reply:
@knadh fixed this PR. Could you review these changes and share your feedback? Thank you.